build/media_source/plg_installer_packageinstaller/js/packageinstaller.es6.js

```diff
@@ -120,6 +120,11 @@ Joomla = window.Joomla || {};
       const file = files[0];
       const data = new FormData();
 
+      if (!file.type) {
```

Comment:
@brianteeman you could be way more restrictive here. The only accepted files for packages are `.zip and .tar` files so something like 

```suggestion
      if (['application/zip', 'application/x-tar'].includes(file.type)) {
```

acts also as a client side validation for the uploads

Reply:
not that simple. tar and gz aree not official mimetype

Reply:
@brianteeman Where is the list of official mimetypes? The [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) does list these mimetypes.

Reply:
You can refer to the mime types that apache is using: https://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types

Reply:
![image](https://user-images.githubusercontent.com/1296369/178339240-774edad0-4c55-438b-9c30-94fa1f140abf.png)

Reply:
There much variants https://superuser.com/a/960710/190987, 
We can stay with just checking that the type actually exist (I hope server doing proper validation already :smile: ).
If we want to validate it on client side, then have to include ALL variants, for each file type supported in upload here.